docs/openapi/lpa-codes-openapi-v1.yml

```diff
@@ -566,3 +583,4 @@ components:
                   x-ray:
                     type: string
                     example: "93c330d4-7d84-4c1b-8fdb-54cec5bfe747"
+
```

Comment:
For the errors, chuck in a 502 for completeness maybe? Not sure if they ever get generated. Optional..

Reply:
I know API G gives 502s if the lambda response is badly formatted, so may be worth adding.
Will deal with errors codes in a future PR :)